### ISSUE TITLE
exp(2*I*pi/3)

### ISSUE THREAD

**Issue by proy87** [sympy/sympy#21284]
The exponent is not simplified to `-1/2+sqrt(3)/2*I`, while there is some code for simplifying such type of exponents.

**Comment by naveensaigit** [comment]
`simplify` can't seem to do it, but `expand` is able to do it.
``` python
>>> from sympy import *
>>> exp(2*I*pi/3)
exp(2*I*pi/3)
>>> (exp(2*I*pi/3)).simplify(complex=True)
exp(2*I*pi/3)
>>> (exp(2*I*pi/3)).expand(complex=True)
-1/2 + sqrt(3)*I/2
```

**Comment by oscarbenjamin** [comment]
> Looking at it now, I feel like `simplify` does the right job since the number in polar form is "simpler".

I agree. Simplification is heuristic and arbitrary. You could also simplify it to `(-1)^(2/3)`.

The best thing is to have functions for specifying what form is actually wanted. For example I should be able to write this in terms of `sin` or `cos` but that doesn't work because it evaluates automatically:
```python
In [19]: exp(2*I*pi/3).rewrite(sin)
Out[19]: 
  1   √3⋅ⅈ
- ─ + ────
  2    2  

In [20]: sin(2*pi/3)
Out[20]: 
√3
──
2 
```
We should have a `to_radical` function for converting expressions into radical form. It could also work with RootOf.

**Comment by proy87** [comment]
`exp(7*pi*I/2)` is evaluated to `-i`. What is the difference between `exp(2*I*pi/3)`?

**Comment by naveensaigit** [comment]
> `exp(7*pi*I/2)` is evaluated to `-i`. What is the difference between `exp(2*I*pi/3)`?

I am not completely sure, but what I observed is that the exponential is evaluated when it is of the form `exp(n*pi*I)` or `exp((2*n+1)*pi*I/2)` as it yields a single value (either entirely real or complex).

**Comment by proy87** [comment]
@naveensaigit , because this is written in the code.
